airflow/www/templates/airflow/task.html

```diff
@@ -39,14 +39,11 @@ <h5>Dependencies Blocking Task From Getting Scheduled</h5>
                 </tr>
             {% endfor %}
         </table>
-        {% if html_code is defined %}
-            {{ html_code|safe }}
-        {% endif %}
```

Comment:
This was never defined inside the `/task` handler, the only place this template is called from.

(And if it was the content would have been duplicated!